HMCL.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.jackhuang.hmcl</id>
```

Comment:
the domain `jackhuang.org` doesn't exist. So this app-id isn't valid. `io.github.huanghongxun.HMCL` would be acceptable since that refers to the github repository.